web-client/src/views/FileDocument/FileEntryOfAppearanceReview.tsx

```diff
@@ -0,0 +1,152 @@
+import { Button } from '../../ustc-ui/Button/Button';
```

Comment:
I know we are removing pa11y right now, so not a great time to ask. Is this new view covered by an accessibility test?

Reply:
Nate and I will write an axe test together in a new PR.